app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :tweet
+  belongs_to :user
+
+  #add association validations to your model
+  validates_associated :tweet, :user
+
+  #User like
+  validates: user_id, uniqueness: true
+  #Liked tweet
+  validates: tweet_id, uniqueness: true
```

Comment:
In this case the validation would have been `validates :tweet_id, uniqueness { scope: :user_id }` otherwise when different users try to like the same tweet is not going to work for one of them

Reply:
Also there is a typo i should be `validates :tweet_id` not `validates: tweet_id`